nautilus_trader/persistence/catalog/parquet.py

```diff
@@ -236,6 +236,7 @@ def write_chunk(
         data_cls: type[Data],
         instrument_id: str | None = None,
         basename_template: str = "part-{i}",
+        write_data_mode: str = "overwrite",
```

Comment:
This naming is explicit, but since we know this is already writing, maybe `mode` would be cleaner?

Reply:
ok I've changed accordingly